game/src/common/mod.rs

```diff
@@ -97,7 +97,7 @@ impl CommonState {
         } else if app.opts.dev {
             Text::from_all(vec![
                 Line("Nothing selected. Hint: "),
-                Line("Ctrl+J").fg(g.style().hotkey_color),
+                Line("Ctrl+J").fg(g.style().text_hotkey_color),
```

Comment:
another rename... trying to do a reverse polish notation thing to group the theme variables.